crypto/src/test/java/net/corda/v5/crypto/SecureHashJavaApiTest.java

```diff
@@ -14,7 +14,7 @@ public void shouldCreateInstance() {
             36, 45, 42, 121, 39, 35, -29, -3, 111, -48, 97, -7, -43, -49, -47, 59, -113, -106, 19, 88, -26, -83, -70, 74
         };
         var cut = SecureHash.parse(str);
-        assertEquals(DigestAlgorithmName.SHA2_384.getName(), cut.getAlgorithm());
+        assertEquals(DigestAlgorithmName.SHA2_384.getName(), cut.algorithm);
```

Comment:
This shouldn't change if we make property private and add getter

Reply:
yes, reverted.